praw/objector.py

```diff
@@ -5,6 +5,16 @@
 class Objector(object):
     """The objector builds :class:`.RedditBase` objects."""
 
+    @staticmethod
+    def fix_dict_keys(d, corrections):
+        """Return a copy of d with keys replaced according to corrections.
+
+        :param d: The dict to be corrected.
+        :param corrections: A dict mapping keys to their replacements.
+
+        """
+        return {corrections.get(k, k): v for k, v in d.items()}
+
```

Comment:
While I like the explicit nature of this approach, I'm concerned that Reddit might add new camel cased attributes. Then each change to make a new attribute snake cased would require a major version bump of PRAW.

The alternative, is that we only show the explicitly allowed attributes, but then we have to release a new version in order for people to get the new attribute support. It'd be nice if it were dynamic.

I think the code from this SO answer will work: http://stackoverflow.com/a/1176023/176978